src/PrivateDns/PrivateDns/ChangeLog.md

```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Rename Update- cmdlets to Set-
```

Comment:
WHat is this part about renaming update cmdlets to set?  That didn't happen in this pr

Reply:
https://github.com/Azure/azure-powershell/pull/8789

its the changes from the above pr, missed adding it to changelog last time.